test/light-driver.test.ts

```diff
@@ -0,0 +1,12 @@
+import test from "ava";
+import uc from "uc-integration-api";
+
+//driver_url field, where we had issues before. Check that the field is not set.
+
+test("Check if driver_url is not defined, so it can be overwritten", (t) => {
+  uc.init("src/light-driver.json");
```

Comment:
This is a heavy weight test and not a unit test:
- the `init` method starts the full WebSocket server and mDNS advertisement.
- the test times out after around 10 seconds.
```console
> npm run build && ava dist/test/**/*.js


> integration-ts-example@0.1.0 build
> tsc && copyfiles "./src/**/*.json" dist/


  ✔ Check if driver_url is not defined, so it can be overwritten
  
  ✘ Timed out while running tests

  Failed to exit when running dist/test/light-driver.test.js

  ─

  1 test passed
```

This test needs to be refactored:
- moved into the uc-integration-api project
- testing the internal `IntegrationAPI.#getDriverUrl` method instead of initializing the full driver.
  - This might involve changing the access modifier from private to protected.

Reply:
I created issue number #31 in the integration-node-library repository. I also replaced this test case with a dummy test.